### DISCUSSION TITLE
Annullamento invoices inviate a SDI

### DISCUSSION THREAD

**Discussion by Mustino2022** [fattureincloud/api#236] [Q&A] [answered]
Buongiorno, 
ho una licenza premium, durante la fase di testing del nostro CRM ho inviato delle fatture elettroniche di prova a SDI tramite le vostre API. Vi chiedo come procedere con l'annullamento perchè sono solo prove per verificare il corretto funzionamento.
Inoltre se è possibile tramite un flag di continuare a inviare fatture elettroniche a SDI indicando che devono essere rifiutate.
Cordiali Saluti

**Comment by valmoz** [comment, marked as answer]
Buongiorno,
purtroppo se sono state inviate al sistema di interscambio non c'è possibilità di annullarle.

A disposizione per ulteriori domande
Mauro

**Comment by Mustino2022** [comment]
Buongiorno, grazie per la risposta,
in questa fase di testing conviene creare la fattura elettronica, effettuare l'invio a SDI e creare una relativa nota di credito.
Nella documentazione delle API per C# SDK è disponibile un esempio di creazione nota di credito?
Altrimenti può condividere un esempio qui nella discussione per la nota di credito che fa riferimento alla fattura già inviata da annullare
Cordiali Saluti

**Reply by emajo** [reply]
Ciao @Mustino2022, l'operazione in questione è molto semplice, basta infatti [fare una get della fattura interessata](https://developers.fattureincloud.it/api-reference#get-/c/-company_id-/issued_documents/-document_id-), e modificare il campo data.type settandolo a `credit_note`, fatto ciò basterà passare il documento alla [chiamata di creazione](https://developers.fattureincloud.it/api-reference#post-/c/-company_id-/issued_documents).

Grazie,
Emanuele.
